src/main/java/org/asamk/signal/commands/AttachmentCommand.java

```diff
@@ -0,0 +1,65 @@
+package org.asamk.signal.commands;
+
+import net.sourceforge.argparse4j.inf.Namespace;
+import net.sourceforge.argparse4j.inf.Subparser;
+
+import org.asamk.signal.commands.exceptions.CommandException;
+import org.asamk.signal.commands.exceptions.UnexpectedErrorException;
+import org.asamk.signal.json.JsonAttachmentData;
+import org.asamk.signal.manager.AttachmentPointer;
+import org.asamk.signal.manager.Manager;
+import org.asamk.signal.output.JsonWriter;
+import org.asamk.signal.output.OutputWriter;
+import org.asamk.signal.output.PlainTextWriter;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Base64;
+
+public class AttachmentCommand implements JsonRpcLocalCommand {
+
+    @Override
+    public String getName() {
+        return "attachment";
```

Comment:
That's a bit unspecific, maybe `getAttachment`?

Reply:
Makes sense. I fixed this. I also changed the class name to match.